### DISCUSSION TITLE
Application remains unanswered

### DISCUSSION THREAD

**Discussion by MohanedElDeebGitHub** [community/community#113184] [GitHub Education] [answered]
### Hi everyone,

I submitted my application a few days ago, and I didn't get any response back. Not sure where I read this - but I thought applications are supposed to be verified in ~1 hour which is why I made this post.

Note: My college is verified, and my docs are in English, not using VPN and my profile matches the credentials.

**Comment by lin20260407** [comment, marked as answer]
If you submitted your application a few days ago and haven't received a response, it's important to stay patient and consider a few potential reasons for the delay. The expected turnaround time for application verification can vary significantly depending on the organization or service you're dealing with. While you might have read about a 1-hour verification time, this could be an ideal scenario rather than a guaranteed timeframe ~
